src/test/scala/scala/concurrent/stm/FlipperSuite.scala

```diff
@@ -228,8 +228,8 @@ class FlipperSuite extends FunSuite {
     }
   }
 
-  def computeSequential(config: Config, P: Array[Ref[Boolean]]): Array[Ref[Int]] = {
-    val A = Array.tabulate[Ref[Int]](config.wordCount) { _ => Ref(0) }
+  def computeSequential(config: Config, P: Array[Ref[Boolean]]): IndexedSeq[Ref[Int]] = {
+    val A = Array.tabulate[Ref[Int]](config.wordCount) { _ => Ref(0) }.toIndexedSeq
```

Comment:
Why can’t we return an `Array` anymore?

Reply:
They are mixing array(mutable) an immutable.Seq. I get the following warning:

```
[warn] /home/gui/scala-stm/src/test/scala/scala/concurrent/stm/FlipperSuite.scala:235:34: method copyArrayToImmutableIndexedSeq in class LowPriorityImplicits2 is deprecated (since 2.13.0): Implicit conversions from Array to immutable.IndexedSeq are implemented by copying; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call
[warn]         (new FlipperTask(config, A, P, true, thread, sync) {
[warn]                                  ^
[warn] /home/gui/scala-stm/src/test/scala/scala/concurrent/stm/FlipperSuite.scala:242:5: method copyArrayToImmutableIndexedSeq in class LowPriorityImplicits2 is deprecated (since 2.13.0): Implicit conversions from Array to immutable.IndexedSeq are implemented by copying; Use the more efficient non-copying ArraySeq.unsafeWrapArray or an explicit toIndexedSeq call
[warn]     A
[warn]     ^
```

Reply:
Why not change `FlipperTask` to take a `collection.Seq` rather than a `collection.immutable.Seq`?